src/DssSpell.sol

```diff
@@ -31,7 +31,7 @@ contract DssSpellAction is DssAction {
     // This should be modified weekly to provide a summary of the actions
```

Comment:
L23 can be removed as not used in the current spell and is referring to previous spell.